Constitution.txt

```diff
@@ -1,490 +1,264 @@
 
 Section 8
-1: The Congress shall have Power To lay and collect Taxes, Duties, Imposts and Excises, to pay the Debts and provide for the common Defence and general Welfare of the United States; but all Duties, Imposts and Excises shall be uniform throughout the United States;
+The Congress shall have Power:
 
-2: To borrow Money on the credit of the United States;
+1:  To lay and collect Taxes, Duties, Imposts, and Excises, on commerce among the several States and with foreign nations; to pay the Debts and provide for the common Defense of these United States; but all Duties, Imposts and Excises shall be uniform throughout these United States; and no Tax, Duty, Impost, or Excise shall apply to transactions occurring wholly within one State, nor shall any resident of any State be compelled by law to participate in a transaction other than the payment of such Taxes, Duties, Imposts, and Excises.
 
-3: To regulate Commerce with foreign Nations, and among the several States, and with the Indian Tribes;
+2: To borrow Money on the credit of these United States, when such credit shall be approved and a payment plan funded by laws passed as described in Article I, Section 7, Part 2;
 
-4: To establish an uniform Rule of Naturalization, and uniform Laws on the subject of Bankruptcies throughout the United States;
+3: To regulate Commerce with foreign Nations, and that which crosses borders among the several States;
 
-5: To coin Money, regulate the Value thereof, and of foreign Coin, and fix the Standard of Weights and Measures;
+4: To establish a uniform Rule of Naturalization, and uniform Laws on the subject of Bankruptcies throughout these United States;
 
-6: To provide for the Punishment of counterfeiting the Securities and current Coin of the United States;
+5: To constitute Tribunals inferior to the supreme Court;
 
-7: To establish Post Offices and post Roads;
+6: To define and punish Piracies and Felonies committed on the high Seas, and Offences against the Law of Nations;
 
-8: To promote the Progress of Science and useful Arts, by securing for limited Times to Authors and Inventors the exclusive Right to their respective Writings and Discoveries;
+7: To declare War, grant Letters of Marque and Reprisal, and make Rules concerning Captures on Land and Water, but when War is declared, no troops or materiel shall issue except from those States whose representatives have unanimously approved the War;
 
-9: To constitute Tribunals inferior to the supreme Court;
+8: To provide and maintain a Navy;
```

Comment:
A Navy is a different beast from an Army. The purpose of the Navy is to operate in international waters (including airspace and outer space); that makes it an international concern, and therefore the purview of the Union and not of an individual State.